### DISCUSSION TITLE
How can I create an app?

### DISCUSSION THREAD

**Discussion by easterncoder** [uvdesk/community-skeleton#639] [Q&A]
How do I create an app so I can add functionality? Is there documentation that I'm not seeing? Thanks and loving it!

**Comment by PeopleInside** [comment]
Hi, what do you mean with "how to create an app"?
If you need add an additional function / app you can read this article: [Additional apps that you can add to the open-source UVdesk helpdesk system](https://forums.uvdesk.com/topic/2010/additional-apps-that-you-can-add-to-the-open-source-uvdesk-helpdesk-system)
